tests/CMakeLists.txt

```diff
@@ -163,6 +163,21 @@ if(GEN_FILES)
             ${CMAKE_CURRENT_SOURCE_DIR}/../include/psa/crypto_extra.h
     )
 
+    add_custom_command(
+        OUTPUT
+            ${CMAKE_CURRENT_SOURCE_DIR}/opt-testcases/tls13-compat.sh
+        WORKING_DIRECTORY
+            ${CMAKE_CURRENT_SOURCE_DIR}/..
+        COMMAND
+            "${MBEDTLS_PYTHON_EXECUTABLE}"
+            "${CMAKE_CURRENT_SOURCE_DIR}/scripts/generate_tls13_compat_tests.py"
+        DEPENDS
+            ${CMAKE_CURRENT_SOURCE_DIR}/../framework/scripts/generate_test_keys.py
+    )
+    add_custom_target(tls13-compat.sh
```

Comment:
`ssl-opt` which is added by https://github.com/Mbed-TLS/mbedtls/pull/9565 should also generate `tls13-compat.sh`. This can only be done once both parts of the code are in — reminder to make a follow-up PR.